bigtable-autoscaler-service.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: bigtable-autoscaler
+  annotations:
+    cloud.google.com/neg: '{"ingress": true}'
+  labels:
+    app: bigtable-autoscaler
+spec:
```

Comment:
This does not require an ingress -- we will never expose this service outside the cluster

Reply:
good point - didn't consider the ingress